src/components/MoneyRequestConfirmationList.js

```diff
@@ -526,7 +526,7 @@ function MoneyRequestConfirmationList(props) {
                             disabled={didConfirm || props.isReadOnly}
                         />
                     )}
-                    {canUseTags && !!tagList && (
+                    {canUseTags && !_.isEmpty(tagList) && (
```

Comment:
I used `isEmpty`  recently for the categories as well. However, there is a feature that our tags and categories could be disabled. It means the app should take into account only enabled ones.